.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,50 @@
+name: Build and Test
+
+# The event triggers are configured as following:
+# - on `master` -> trigger the workflow on every push
+# - on any pull request -> trigger the workflow
+# This is to avoid running the workflow twice on pull requests.
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 13.x]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: yarn cache directory
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Setup node_modules cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install
+        run: yarn install --frozen-lockfile
+      - name: Build
+        run: npm run build
+      - name: Lint
+        run: npm run lint
+      - name: Test
+        env:
+          TEST_BROWSERSTACK: ${{ startsWith(matrix.node-version, '13') }}
+          TEST_PROBE_ONLY: ${{ github.ref != 'refs/heads/master' }}
+          BS_USERNAME: ${{ secrets.BS_USERNAME }}
```

Comment:
@cure53 I don't see the `Settings`-tab under the repository. Can you add these as secrets on GitHub. You can find them here. I suppose we have to generate a new set on BrowserStack for the GitHub Actions. Still one we got "rid of" TravisCI we can remove the old ones.

![CleanShot 2020-04-19 at 11 08 53@2x](https://user-images.githubusercontent.com/1877073/79683978-33f4a900-822e-11ea-8097-c435d69d2aa9.png)

Reply:
What values to initialize them with?

Reply:
Added them, two set to `true` and one to the username accordingly